generator/src/test/java/com/scottlogic/deg/generator/cucumber/OfLength.feature

```diff
@@ -560,20 +488,18 @@ Feature: User can specify the length of generated string data using 'ofLength'
       | foo  |
       | "22" |
 
-
-  @ignore
-  Scenario: Running an 'ofLength' request alongside a contradicting shorterThan constraint should fail with an error message
+  Scenario: Running an 'ofLength' request alongside a contradicting shorterThan constraint should produce null
     Given foo is of length 2
     And foo is shorter than 2
     And foo is in set:
       | "1"  |
       | "a"  |
       | "22" |
-    Then I am presented with an error message
-    And no data is created
-
+    Then the following data should be generated:
+      | foo  |
+      | null |
 
-  @ignore
+ @ignore
```

Comment:
Can we raise a bug on this or link to an existing?

Reply:
This will be done as part of ticket #501